ISeriesProvider/DB2iSeriesSchemaProvider.cs

```diff
@@ -30,8 +39,9 @@ protected override List<ColumnInfo> GetColumns(DataConnection dataConnection)
 				, Table_Name
 				, Table_Schema
 				, Column_Name
-				From QSYS2/SYSCOLUMNS

```

Comment:
DB2Connect doesn't support '/' for delimiter, refactored delimiter to a function from naming convention